XRTK-Core/Packages/com.xrtk.core/Services/PlatformSystem/Platforms/EditorPlatform.cs

```diff
@@ -24,16 +25,12 @@ public EditorPlatform(string name, uint priority)
         }
 
         /// <inheritdoc />
-        public bool IsActive
-        {
-            get
-            {
-#if UNITY_EDITOR
-                return true;
-#else
-                return false;
-#endif
-            }
-        }
+        public override string Name => "Editor";
+
+        /// <inheritdoc />
+        public override uint Priority => 0;
+
+        /// <inheritdoc />
+        public bool IsActive => Application.isEditor;
```

Comment:
This method is a lot better, but can we rename "IsActive" to "IsAvailable", as we aren't checking if it's active, rather that it exists to be used